test/sparql-proxy.js

```diff
@@ -228,4 +228,27 @@ describe('sparql-proxy', () => {
       .get('/query?query=' + encodeURIComponent(query))
       .expect(502)
   })
+
+  it('should not crash even if redis is not recheable', async () => {
```

Comment:
For this one, we could have two tests:

Query with caching functional and enabled. 
	1. First query hits the (nock) endpoint
	2. Second query hits the cache

Query with caching non-functional and enabled. 
	1. First query hits the (nock) endpoint
	2. Second query hits the (nock) endpoint

Reply:
Done! I added a third request to make sure it stays stable.